packages/taxdat/R/plot_case_raster_admin1_shp.R

```diff
@@ -0,0 +1,41 @@
+#' @export
+#' @name plot_case_raster_admin_shp
+#' @param config
+#' @param cholera_directory
+#' @param cache
+plot_case_raster_admin_shp <- function(cache, cholera_directory, config) {
```

Comment:
I think we should make the function name match the name of the R script. Could you align these and rerun devtools::document on the package?